zproject/prod_settings_template.py

```diff
@@ -279,6 +279,10 @@
     "emailAddress": ZULIP_ADMINISTRATOR,
 }
 
+# Set this to True to enforce that any configured IdP needs to specify
+# the limit_to_subdomains setting to be considered valid.
+SAML_REQUIRE_LIMIT_TO_SUBDOMAINS = False
```

Comment:
Let's move the comment to `default_settings.py` and remove this from `prod_settings_template.py`.  I don't think we want this to be part of the documented settings for clutter-management reasons.